apps/api/src/app/feature-flags/use-cases/get-feature-flag/get-feature-flag.use-case.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { GetFeatureFlagCommand } from './get-feature-flag.command';
+
+import { FeatureFlagsService } from '../../services';
+
+@Injectable()
+export class GetFeatureFlag {
+  constructor(private featureFlagsService: FeatureFlagsService) {}
+
+  async execute(command: GetFeatureFlagCommand): Promise<void> {
+    const { key, environmentId, organizationId, userId } = command;
+
+    const context = {
+      environmentId,
+      organizationId,
+      userId,
+    };
+
+    // TODO: The type needs to be dynamic. String so far by default
+    await this.featureFlagsService.get<string>(key, context, '');
```

Comment:
you don't have to specify generic type `<string>` here as it would be inferred from the default value

Reply:
Didn't know this one. Thanks!